tests/specs/send/send-stx.spec.ts

```diff
@@ -128,5 +130,20 @@ test.describe('send stx', () => {
       const details = await sendPage.confirmationDetails.allInnerTexts();
       test.expect(details).toBeTruthy();
     });
+
+    test('that address on preview mathes input one', async ({ sendPage }) => {
+      await sendPage.amountInput.fill('0.000001');
+      await sendPage.recipientInput.fill(TEST_ACCOUNT_2_STX_ADDRESS);
+      await new Promise(res => {
```

Comment:
don't like this, but without it test fails with timeout error, not sure yet how to fix this, also seems test above is false positive as after preview btn click page is not changed

Reply:
seems problem was that fees value was not defined in the moment when preview btn was clicked and it led to tests failing. fixed now